### ISSUE TITLE
Show ahead-behind info and perhaps "dirty" flag

### ISSUE THREAD

**Issue by krlmlr** [roeme/shgit#3]
I think the `+x-y` or `=` indicators that `git-sh` had are helpful to understand quickly if a push/pull action is needed. How many syscalls do we need for:

```sh
git for-each-ref --format="%(upstream:track)%(upstream:trackshort)" refs/heads/master
```

The "dirty" indicator would be nice to have too.

**Comment by roeme** [comment]
Agree, and already planned to add the indicators. For resource-constrained environments as described in the README, they should be configurable.

**Comment by krlmlr** [comment]
Closed in favor of #4.
